statvar_imports/us_college_ipeds/preprocess.py

```diff
@@ -0,0 +1,36 @@
+import os
+import pandas as pd
+from absl import logging
+
+def clean_csv(file_path):
+    with open(file_path, 'r') as f:
+        lines = f.readlines()
+
+    start_index = -1
+    for i, line in enumerate(lines):
+        if line.strip().startswith('Year'):
+            start_index = i
+            break
+    
+    if start_index != -1:
+        cleaned_content = lines[start_index:]
+        with open(file_path, 'w') as f:
+            f.writelines(cleaned_content)
```

Comment:
We generally avoid modifying the original source files so that we can trace any issues back easily. Can we write the content to new files and have statvar processor work on them instead?